engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/consumerrecord/ConsumerRecordVariableProvider.scala

```diff
@@ -0,0 +1,42 @@
+package pl.touk.nussknacker.engine.kafka.consumerrecord
+
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.api.context.{OutputVar, ValidationContext}
+import pl.touk.nussknacker.engine.api.typed.typing.Typed
+import pl.touk.nussknacker.engine.api.{Context, ContextInterpreter}
+import pl.touk.nussknacker.engine.flink.util.context.{InitContextFunction, VariableProvider}
+
+import scala.reflect.ClassTag
+
+class ConsumerRecordVariableProvider[K: ClassTag, V: ClassTag] extends VariableProvider[DeserializedConsumerRecord[K,V]] {
```

Comment:
Regards to name, IMO it is more important that it is Kafka... then ConsumerRecord...

Reply:
Fixed. Renamed to `KafkaContextInitializer`.